handler/src/main/java/io/netty/handler/pcap/PcapWriteHandler.java

```diff
@@ -579,4 +630,98 @@ public void close() throws IOException {
             logger.debug("PcapWriterHandler is now closed");
         }
     }
+
+    private enum ChannelType {
+        TCP, UDP
+    }
+
+    /**
+     * Builder for {@link PcapWriteHandler}.
+     */
+    public static final class Builder {
+        boolean captureZeroByte;
+        boolean writePcapGlobalHeader = true;
+
+        ChannelType channelType;
+        InetSocketAddress initiatiorAddr;
+        InetSocketAddress handlerAddr;
+        boolean isServerPipeline;
+
+        Builder() {
```

Comment:
```suggestion
        private Builder() {
```

Reply:
i made this package-private to save on bridge methods, which may be overkill (same for the fields). should i make them all private?